pkg/capacity/capacity.go

```diff
@@ -99,3 +103,19 @@ func (r *ResourceOracle) Disks() (d Disks, err error) {
 
 	return
 }
+
+// IsVM detect if machine is virtual
+func (r *ResourceOracle) IsVM() (bool, error) {
```

Comment:
After reflection, instead of returning just a boolean, I would prefer we return the list of thing virt-what have found.
So we can report the result to BCDB.